examples/array-datatype/AppManifest.json

```diff
@@ -1,40 +1,18 @@
 [
     {
         "Name": "array-datatype",
-        "Port": 50008,
-        "DAPR_GRPC_PORT": 50001,
```

Comment:
Is it already clarified where we put these values instead?

Reply:
As discussed, this is hardcoded for now in the devenv-runtime scripts and will be solved later with the middleware-selection